src/compiler/transformers/decorators-to-static/convert-decorators.ts

```diff
@@ -42,19 +43,13 @@ const visitClassDeclaration = (
   typeChecker: ts.TypeChecker,
   classNode: ts.ClassDeclaration
 ) => {
-  if (!classNode.decorators) {
-    return classNode;
-  }
-
-  const componentDecorator = classNode.decorators.find(isDecoratorNamed('Component'));
+  const componentDecorator = retrieveTsDecorators(classNode)?.find(isDecoratorNamed('Component'));
```

Comment:
The use of optional chaining here allows us to remove the first conditional statement, checking if `classNode.decorators` is truthy